maven-plugin-plugin/pom.xml

```diff
@@ -350,7 +350,8 @@
               <settingsFile>src/it/settings.xml</settingsFile>
               <streamLogsOnFailures>true</streamLogsOnFailures>
               <filterProperties>
-                <sitePluginVersion>3.3</sitePluginVersion>
+                <sitePluginVersion>3.11.0</sitePluginVersion>
+                <projectInfoReportsPlugin>3.2.2</projectInfoReportsPlugin>
```

Comment:
Why not use the same version like in project?
Do we want later use another version in tests then in project?
I afraid that next we update version in project but not for tests.
And how to managed version if we have plugin version from parent?

Reply:
In most cases reactor plugins and IT plugins are decoupled. Look at other projects. Moreover, I did not want to rewrite the entire thing. You have valid concerns, but they need to be addressed on a global level for all components.

Reply:
ok, agreed